op-e2e/e2eutils/blobs.go

```diff
@@ -0,0 +1,48 @@
+package e2eutils
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+// BlobsStore is a simple in-memory store of blobs, for testing purposes
+type BlobsStore struct {
+	blobs map[uint64]map[common.Hash]*eth.Blob
+}
+
+func NewBlobStore() *BlobsStore {
+	return &BlobsStore{blobs: make(map[uint64]map[common.Hash]*eth.Blob)}
+}
+
+func (store *BlobsStore) StoreBlob(time uint64, versionedHash common.Hash, blob *eth.Blob) {
+	m, ok := store.blobs[time]
+	if !ok {
+		m = make(map[common.Hash]*eth.Blob)
+		store.blobs[time] = m
+	}
+	m[versionedHash] = blob
+}
+
+func (store *BlobsStore) GetBlobs(ctx context.Context, ref eth.L1BlockRef, hashes []eth.IndexedBlobHash) ([]*eth.Blob, error) {
```

Comment:
Is `GetBlobs` explicitly unsorted?

Reply:
isn't it sorted like `hashes`? It's build up in a loop over `hashes`.

Reply:
oh yea it is. It matches the specs